releasenotes/notes/fix-UCC-and-fully-occupied-MOs-c8fdf252a0607395.yaml

```
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Allow for building UCC
+    (:class:`qiskit_nature.second_q.circuit.library.ucc`) ansatze for systems
```

Actually, I just noticed that this probably won't show up correctly unless we change it like so:
```suggestion
    Allow for building :class:`.UCC` ansatze for systems
```